AC3.2Unit2FinalAssessment/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,174 @@
+        </scene>
+        <!--Crayon View Controller-->
+        <scene sceneID="hCh-mr-ILs">
+            <objects>
+                <viewController id="vun-Gh-iIx" customClass="CrayonViewController" customModule="AC3_2Unit2FinalAssessment" customModuleProvider="target" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="VjZ-5i-8zb"/>
+                        <viewControllerLayoutGuide type="bottom" id="pjT-4p-KVv"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="fjK-7Z-22C">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                    </view>
+                    <navigationItem key="navigationItem" id="TwF-4x-aCS">
+                        <barButtonItem key="backBarButtonItem" title="Back" id="FE0-6Q-5e3"/>
+                    </navigationItem>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="fWa-II-RMe" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="1153" y="2"/>
+        </scene>
+        <!--Tab Bar Controller-->
+        <scene sceneID="bm8-pa-ur9">
+            <objects>
+                <tabBarController id="QiK-Y2-Aq3" sceneMemberID="viewController">
+                    <tabBar key="tabBar" contentMode="scaleToFill" id="Eye-Eh-aJi">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="49"/>
+                        <autoresizingMask key="autoresizingMask"/>
+                        <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                    </tabBar>
+                    <connections>
+                        <segue destination="Py9-Xs-P4U" kind="relationship" relationship="viewControllers" id="yNj-Or-apa"/>
+                        <segue destination="6gO-ho-Eqo" kind="relationship" relationship="viewControllers" id="seU-fs-38b"/>
+                        <segue destination="OjY-VF-H0x" kind="relationship" relationship="viewControllers" id="YXs-WD-AHB"/>
+                    </connections>
+                </tabBarController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="qgm-Qh-nLi" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="-1174" y="3"/>
+        </scene>
+        <!--Item -->
+        <scene sceneID="s3N-sl-bRG">
+            <objects>
+                <navigationController automaticallyAdjustsScrollViewInsets="NO" id="Py9-Xs-P4U" sceneMemberID="viewController">
```

Comment:
Your tab bar should connect to your navigation control ONLY, not the tableView controller.